.github/workflows/gates_report_brillig.yml

```
@@ -74,17 +74,19 @@ jobs:
 
       - name: Compare Brillig bytecode size reports
         id: brillig_bytecode_diff
-        uses: noir-lang/noir-gates-diff@cd8721ebe0ad9a67889c601063cccc33a8ca7f32
+        uses: noir-lang/noir-gates-diff@79010d656ae50e83a28e5139ed3850b8cc9873d1
```

```suggestion
        uses: noir-lang/noir-gates-diff@3fb844067b25d1b59727ea600b614503b33503f4
```

Updating to point at main branch.